src/hydro/srcterms/tabular_cooling.cpp

```diff
@@ -460,7 +468,11 @@ void TabularCooling::SubcyclingFixedIntSrcTerm(MeshData<Real> *md, const Real dt
           sub_iter++;
         }
 
-        PARTHENON_REQUIRE(internal_e > internal_e_floor, "cooled below floor");
+        //PARTHENON_REQUIRE(internal_e > internal_e_floor, "cooled below floor");
+        //HACK
+        if(internal_e < internal_e_floor){
+          internal_e = internal_e_floor;
+        }
```

Comment:
Should this remain? Or in other words, should this be safety check (for this subcycling) or an early "fix" what would otherwise later be triggered in `ConsToPrim`?

Reply:
I saw issues for negative energies before this line was added. If that fix isn't already in `ConsToPrim` but just wasn't working, we could put it there as long as other source terms after this don't depend on `Cons` or `Prim`

Reply:
This should be addressed correctly in #28